staging/src/k8s.io/kubectl/pkg/cmd/describe/describe.go

```diff
@@ -69,6 +69,32 @@ var (
 		kubectl describe pods frontend`))
 )
 
+// DescribeFlags directly reflect the information that CLI is gathering via flags. They will be converted to Options,
+// which reflect the runtime requirements for the command.
+type DescribeFlags struct {
+	Factory           cmdutil.Factory
+	Selector          string
+	AllNamespaces     bool
+	FilenameOptions   *resource.FilenameOptions
+	DescriberSettings *describe.DescriberSettings
+	genericclioptions.IOStreams
+}
+
+// NewDescribeFlags returns a default DescribeFlags
+func NewDescribeFlags(f cmdutil.Factory, streams genericclioptions.IOStreams) *DescribeFlags {
+	return &DescribeFlags{
+		Factory:         f,
+		Selector:        "",
```

Comment:
Maybe we can remove these `Selector` and `AllNamespaces` fields setting, their default is already empty and false?

Reply:
yeah make sense 👍🏻